service/src/test/java/bio/terra/drshub/controllers/DrsHubApiControllerTest.java

```diff
@@ -144,6 +147,8 @@ void testCallsCorrectEndpointsWhenOnlyAccessUrlRequestedWithPassportsUsingFallba
 
     // need an extra verify because nothing in the mock cares that bearer token is set or not
     verify(drsApi).setBearerToken(TEST_BOND_SA_TOKEN);
+    verify(drsApi, never())
```

Comment:
I'm not sure this verification gives us full peace of mind -- could you expand it so that we're verifying that `postAccessURL` is never called with any inputs under this test set up?

Would you also mind adding a comment for this verification too?  I wish `verify` let us add a reason to the call!

Reply:
PS. While you're in the neighborhood, it would be nice to add another test to handle the case where passport is a supported means of auth, but `postAccessURL` throws an exception, to make sure that we gracefully fall back in that case too.  Maybe there's a test that already does that!